### DISCUSSION TITLE
Execution of lesson 1 contract (SimpleStorage on Remix) fails

### DISCUSSION THREAD

**Discussion by pythoulon** [smartcontractkit/full-blockchain-solidity-course-py#13] [Q&A] [answered]
After many tries, I resorted to copy the code directly from the GitHub repository https://github.com/PatrickAlphaC/simple_storage to make sure I didn't have anything fishy in my code. I still have the following problem:

1. I type a string and a number in the "addPerson" box (eg "me", 5)
2. I click on the addPerson button
3. I click on the people button to view the results, and I get this error message: `call to SimpleStorage.people errored: Error encoding arguments: Error: invalid BigNumber string (argument="value", value="", code=INVALID_ARGUMENT, version=bignumber/5.4.1)`

It looks like the entry was never added to people array, nor to the nameToFavoriteNumber mapping for that matter. I've been using different versions of the compiler, with the same results. I compile and deploy OK on the Javascript VM.

Any idea what might be wrong ?
Thanks.

**Comment by AlbertoSinigaglia** [comment, marked as answer]
Just tested, it is working for me:

<img width="1440" alt="Schermata 2021-09-13 alle 16 59 18" src="https://user-images.githubusercontent.com/25763924/133107520-8460851c-1242-4a9b-93d0-d5831fc2710d.png">

**Reply by pythoulon** [reply]
Lucky guy... Mine keeps failing. See error in the log pane.

![Snap15](https://user-images.githubusercontent.com/11643101/133109158-14efa15f-b4a8-4102-a79f-d7cdc1c19efd.png)

**Reply by AlbertoSinigaglia** [reply]
if you are clicking `people` you have to set the index first on the right

**Reply by pythoulon** [reply]
Argh, silly me. Thanks a lot!

**Reply by danielmacuare** [reply]
I spent 30 minutes doing the same mistake. Thanks, both, this helped!
